tools/Google.Cloud.Tools.DocUploader/Google.Cloud.Tools.DocUploader.csproj

```diff
@@ -1,10 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="../DotnetToolProperties.xml" />
+
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+
+    <Version>0.1.0</Version>
+    <Description>Tool used by Google .NET release processes for documentation. While there is nothing "secret" in this package, it is unlikely to be useful to other developers.</Description>
```

Comment:
nit: Maybe adding a note saying, "we just publish this for convenience" to make emphasis. Here and in the other one.

Reply:
Yup, will do.